### DISCUSSION TITLE
Album Settings not visible

### DISCUSSION THREAD

**Discussion by tinohager** [LycheeOrg/Lychee#2612] [Bugs]
### Lychee version

v6.0.0

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

If I have opened an album containing several photos and then scroll down and open the album settings using the arrow, these are not visible.

### Steps to reproduce the issue

1. Go to an Album
2. Scroll down
3. Open the Album Settings

### Diagnostics [REQUIRED]

```
Errors
warning
APP_URL is still set to default, this will break access to all your images
info
Latest version of PHP is 8.3
warning
Default timezone not properly set; you might experience strange results when importing photos without explicit EXIF timezone
info
git (software) is not available.
error
Photo without Original found -- IMG_4098.JPG in Test2
error
Photo without Original found -- IMG_4161.JPG in Test2
error
Photo without Original found -- IMG_4145.JPG in Test1
error
Photo without Original found -- IMG_4151.JPG in Test1
info
Found 49 small that could be generated.
info
Found 46 small2x that could be generated.
info
Found 59 medium that could be generated.
info
Found 59 medium2x that could be generated.
info
Have you considered supporting Lychee? :)
Info
Lychee Version (tag):                      v6.0.0 (d25cd64) -- Up to date (-7.6579147640791E-11 years ago).
DB Version:                                6.0.0
composer install:                          --no-dev
APP_ENV:                                   local
APP_DEBUG:                                 false
APP_URL:                                   default
APP_DIR:                                   default
LOG_VIEWER_ENABLED:                        true
VUEJS_ENABLED:                             true
PHOTO_PIPES:                               true
System:                                    Linux
PHP Version:                               8.2.24
PHP User agent:                            Lychee/4 (https://lycheeorg.github.io/)
Timezone:                                  UTC
Max uploaded file size:                    100M
Max post size:                             100M
Chunk size:                                25.60 MB
Max execution time:                        3600
MySQL Version:                             10.11.9-MariaDB-ubu2204
exec() Available:                          yes
Imagick Available:                         1
Imagick Enabled:                           1
Imagick Version:                           1691
GD Version:                                2.3.3
Number of foreign key:                     12 found.
```

### Browser & System [REQUIRED]

chrome, docker:latest

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment]
I can make it scroll automatically up for the settings to be visible.
I decided against to avoid "surprising" the user.

**Comment by tinohager** [comment]
When I click on the button, I find it strange when nothing happens. Otherwise the button should only be visible when you are at the top.

But I would suggest that it is always visible with css style position: fixed; as long as it is open.

**Reply by ildyria** [reply]
The problem with `fixed` is that it prevents scrolling, and it could happen that the settings collapse is bigger than screen height. Not so keen in added another scroll bar in those settings as you can imagine :)

**Reply by tinohager** [reply]
Then I would suggest setting the scroll position to 0 when clicking on the arrow. Then the dialog is also visible.

**Comment by ildyria** [comment]
fixed.
